storage/objects/main.go

```diff
@@ -424,6 +426,75 @@ func downloadUsingRequesterPays(client *storage.Client, object, bucketName, loca
 	return nil
 }
 
+func generateV4GetObjectSignedUrl(client *storage.Client, bucketName, objectName, serviceAccount string) (string, error) {
```

Comment:
Each function should create its own client, rather than have it passed as an argument. It might be better to stay consistent within the same set of samples, but I'm not sure.

Reply:
I'd rather keep the existing format without changing the format unless it's required.

Reply:
See https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#initialize-clients-and-services-in-every-sample (new canonical sample style guide).

Eventually, all samples should match this style. But, if you want to be internally consistent for these samples, that works too.

Reply:
Thanks for clarifying! 

I want to follow the format that exists in these samples to be consistent.